locales/de.json

```diff
@@ -2484,7 +2484,7 @@
 		},
 		"oauth2": {
 			"headline": {
-				"loginWithSchoolCloud": "Login mit {{title}}"
+				"loginWithSchoolCloud": "Über {{title}} anmelden"
```

Comment:
this is already used in https://github.com/hpi-schul-cloud/schulcloud-client/blob/060ca6138b5df464c944489492ef2b7c05c6ded8/controllers/oauth2.js#L73 (oauth flow for bettermarks) why changing that and not use a new term?